### PR TITLE
feat(backButton): added back button

### PR DESCRIPTION
**Pull request by k0malSharma**
**Functionality:**

Back button to avoid reverting to main page every time

**Solution:**

Added a back button on the navbar of each component.

**Risk level:**

- [X] High
- [ ] Medium
- [ ] Low

**How to test:**

1. ng serve
2. By clicking on 'X' (close button), you will be redirected to the previous component.
image

### CODE REVIEW

File: worktrolly-ui/src/app/body/task-details/edit-page/edit-page.component.ts [hunk 1]
```diff
@@ -48,4 +49,8 @@ export class EditPageComponent implements OnInit {
     this.editTaskCompleted.emit({ completed: true });
   }
 
+  backToTaskDetails(){
+    window.location.reload();
+  }
+
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/body/task-details/edit-page/edit-page.component.ts:55]: don't use the reload method here.

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]

**Review by shivamgoyal7 — APPROVED** [review]
Ready to merge

File: worktrolly-ui/.vs/ProjectSettings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (viv9k) [inline worktrolly-ui/.vs/ProjectSettings.json:3]: Please revert these changes

File: worktrolly-ui/.vs/VSWorkspaceState.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\src",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (viv9k) [inline worktrolly-ui/.vs/VSWorkspaceState.json:7]: Please revert these changes

**Review by viv9k — CHANGES_REQUESTED** [review]
Please revert all changes in .vs folder

File: worktrolly-ui/.vs/VSWorkspaceState.json (hunk 3 above)
Comment (viv9k) [reply worktrolly-ui/.vs/VSWorkspaceState.json:7]: also resolve merge conflict

**Comment by shivamgoyal7** [comment]
solve the conflict

**Comment by shivamgoyal7** [comment]
attach screenshot

**Review by shivamgoyal7 — APPROVED** [review]
Ship it

File: worktrolly-ui/src/app/body/task-details/edit-page/edit-page.component.ts [hunk 4]
```diff
@@ -17,7 +18,7 @@ export class EditPageComponent implements OnInit {
   editTask: Tasks
   previousSprintId: number
 
-  constructor(private functions: AngularFireFunctions) { }
+  constructor(private functions: AngularFireFunctions, private router: Router) { }
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/task-details/edit-page/edit-page.component.ts:21]: router is not needed here

**Review by viv9k — CHANGES_REQUESTED** [review]
please do the required changes asap

**Comment by viv9k** [comment]
Please resolve merge conflict also

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**